packages/esm-ward-app/src/ward-workspace/patient-transfer-bed-swap/patient-transfer-swap.workspace.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { getGlobalStore, useFeatureFlag, type DefaultWorkspaceProps } from '@openmrs/esm-framework';
```

Comment:
We're not using either `getGlobalStore` or the `DefaultWorkspaceProps` type import.